### ISSUE TITLE
a lot of illegal instructions on linux aarch64

### ISSUE THREAD

**Issue by ghost** [PSI-Rockin/DobieStation#475]
Hi! I would like to know if there is something runnable (tho not playable), bc even on full interpreter not even the bios want to launch on my rk3399 linux aarch64 test from master. best regards.

**Comment by Ziemas** [comment]
The GS JIT which is x86 is enabled by default at compile time, you can try removing the `#define GS_JIT` line from `gsthread.cpp`. I don't know if there is anything blocking it other than that.

**Comment by mrcmunir** [comment]
The main problem is not JIT its in Interpreter  total broken too  In the past its working on intepreter i tested in early days run 3-6fps speed on jetson nano .
![Screenshot from 2021-09-05 17-14-18](https://user-images.githubusercontent.com/1274014/132132083-7a41565a-0a36-4c9b-84ec-14bcbe1c5635.png)
Edit : @Ziemas confirmed Remove #define GS_JIT prevent this crash but i think defined GS_JIT never will be affected if you change to Interpreter 

Maybe the workarround will be for now .
```
if defined(__x86_64__) || defined(__amd64__) || defined(__x86_64) || defined(_M_AMD64)
    /*
     * both __x86_64__ and __amd64__ are defined by gcc/clang
     * __x86_64 defined by sun compiler on opensolaris at least
     * _M_AMD64 defined by MS compiler
     */
    #define GS_JIT
#endif
```

**Comment by seisdr** [comment]
You can build dobie on arm64??

**Comment by mrcmunir** [comment]
@seisdr Yes but its slow due lack JIT running 2-6fps or so with the actual code 
You needed remove #define GS_JIT in gsthread.cpp or change defined to x86 ARCH only . 
As I commented in my previous comment.

**Comment by seisdr** [comment]
Is it  software rendering?

**Comment by mrcmunir** [comment]
> Is it software rendering?

Yes this that in the actual stage so not too fps on CPU limited 
Here https://github.com/PSI-Rockin/DobieStation/issues/77 they talk about Opengl is 3.2 and Vulkan and extensions of Opengl 4.x but it is from 2019. 
Surely in the future it will be great considered when the base of the emulator core is more advanced.
